### ISSUE TITLE
upgrade to 1.6

### ISSUE THREAD

**Issue by pswenson** [coreos/coreos-kubernetes#881]
Are there plans to upgrade to 1.6?

**Comment by robszumski** [comment]
There are no plans to update this to 1.6. That support, testing and docs is being done in the open-source Tectonic installer. It gets you a Terraform-based Kubernetes installation. It is also open source, uses upstream Kubernetes and can be easily customized.

**Comment by robszumski** [comment]
A note should be added to these docs shortly, the ones on the web have a note already.

**Comment by pswenson** [comment]
this the project you are talking about?  https://github.com/coreos/tectonic-installer

**Comment by robszumski** [comment]
@pswenson Yes, that is correct. Apologies for not linking to it, I have no idea how that slipped my mind.

**Comment by timm088** [comment]
Shame, I quite liked the chance to have a K8s single-node vagrant setup, not sure that's easily available with Tectonic?

**Comment by robszumski** [comment]
@timm088 Tracking this over at https://github.com/coreos/tectonic-forum/issues/120, this is something that would make sense for Tectonic to ship/maintain. Subscribe over there for updates.

**Comment by rbjorklin** [comment]
@pswenson @timm088 I spent some time and created a fairly significant pull request that updates Kubernetes to 1.7.3 and all it's dependencies, view it [here](https://github.com/coreos/coreos-kubernetes/pull/902). It's a vagrant multi-host setup though, I hope that's not a blocker for you!
